experiments/experiments-impl/src/main/java/com/duckduckgo/experiments/impl/ExperimentVariantRepository.kt

```diff
@@ -40,4 +44,10 @@ class ExperimentVariantRepositoryImpl @Inject constructor(
     override fun getActiveVariants(): List<ExperimentVariantEntity> {
         return experimentVariantDao.variants().filter { it.weight != null && it.weight > 0.0 }.toList()
     }
+
+    override fun variantConfigDownloaded() {
+        experimentsDataStore.variantManagerConfigDownloaded = true
+    }
+
+    override fun isVariantManagerConfigReady(): Boolean = experimentsDataStore.variantManagerConfigDownloaded
```

Comment:
Why do we need `isVariantManagerConfigReady` ?? If we don't have a variant we just fallback to default.

Reply:
Outdated, this was removed